go.mod

```
@@ -19,7 +19,7 @@ require (
 	github.com/coreos/go-oidc/v3 v3.9.0
 	github.com/doublerebel/bellows v0.0.0-20160303004610-f177d92a03d3
 	github.com/evanphx/json-patch v5.8.0+incompatible
-	github.com/expr-lang/expr v1.16.0
+	github.com/expr-lang/expr v1.16.1
```

```suggestion
	github.com/expr-lang/expr v1.16.9
```

Should probably just bump to latest patch for extra bug fixes etc

---

Good point, just updated the PR to upgrade it to `v1.16.9`